lib/ansible/modules/cloud/kubevirt/kubevirt_pvc.py

```diff
@@ -437,7 +437,7 @@ def execute_module(self):
             spec['volumeName'] = self.params.get('volume_name')
 
         # 'resource_definition:' has lower priority than module parameters
-        definition = KubeVirtRawModule.merge_dicts(self.resource_definitions[0], definition)
+        definition = dict(KubeVirtRawModule.merge_dicts(definition, self.resource_definitions[0]))
```

Comment:
You don't need to add this extra call ```dict()``` call.  (Nor is it needed in the devel version either).  The current version of merge_dicts in both branches returns a new dictionary so there's no need to turn it into a dictionary in the caller.

Since this 2.8 branch didn't have a dict() call to begin with, I woiuldn't add it now (and you should try to merge removal of the extraneous dict() call to devel too):

```suggestion
        definition = KubeVirtRawModule.merge_dicts(definition, self.resource_definitions[0])
```
```